pkg/logs/input/journald/tailer.go

```diff
@@ -236,10 +240,18 @@ var applicationKeys = []string{
 }
 
 // getApplicationName returns the name of the application from where the entry is from.
-func (t *Tailer) getApplicationName(entry *sdjournal.JournalEntry) string {
+func (t *Tailer) getApplicationName(entry *sdjournal.JournalEntry, tags []string) string {
 	if t.isContainerEntry(entry) {
-		return "docker"
+		if t.source.Config.ContainerMode {
```

Comment:
Currently it always return "docker", and it was requested to add a parameter to enable new behaviour to keep backward compatibility.

Reply:
I think we don't need this config parameter and we always want to use the short image name when we can, so the code would look something like:
```
if t.isContainerEntry(entry) {
	if shortName, found := getDockerImageShortName(t.getContainerID(entry), tags); found {
		return shortName
	}

	return defaultApplicationName
}
```

Reply:
@NBParis What is your opinion about this one?

Reply:
We actually do because of backward compatibility.

If we were to change this it might be a breaking change for customers that have already pipelines defined on the existing source and service value.

So let's keep the parameter but let's add it everywhere in the documentation so that when people copy paste the examples they have it.

Reply:
sounds good 👍, resolving this conversation